jekyll/_cci2/releases/configure-your-kubernetes-components.adoc

```diff
@@ -0,0 +1,74 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Configure your Kubernetes components - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to configure your Kubernetes components for visibility and control on the releases dashboard.
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**
+
+Visualise and control deployments to your Kubernetes cluster with CircleCI release environments. The steps outlined on this page guide you to set up a release environment and install the CircleCI release agent into your Kubernetes cluster.
+
+[#introduction]
+== Introduction
+
+In this tutorial you will configure your Kubernetes components, adding labels and feature flags to enable visibility and control over your deployments, and access to the following features:
```

Comment:
Suggestion: We might want to avoid referring to "feature flags" since that's a separate thing we may offer in the future as a product, and these are more of "settings" specified via K8s annotations.
```suggestion
In this tutorial you will configure your Kubernetes components, adding labels and annotations to enable visibility and control over your deployments, and access to the following features:
```